src/changelog/2024/09/notifications-bulk-actions.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Managing Notifications
+description: "Interact with notifications"
+date: 2024-09-25 10:00:00.0
```

Comment:
Why is this backdated?

Reply:
it slipped through the cracks, should be fixed now